roles/update/templates/workload_launch.sh.j2

```diff
@@ -394,9 +499,18 @@ if [[ "${MODE}" == "cleanup" ]]; then
 fi
 
 if [[ "${MODE}" == "sanity" ]]; then
+    trap cleanup_on_exit EXIT
```

Comment:
Why trap command here and next mode too?

Reply:
Sanity and sanifyfast mode, are special modes where vm are created and then destroyed.  sanityfast just has lower timeout values.

trap EXIT is a nice bash signal extension to make sure the cleanup code run whatever happen during deployment, either an error or a normal exit.

The code then doesn't have to be cluttered with conditional on every command to check if it succeed or not.  We are just certain that cleanup will try to run no matter what.

Reply:
thank you for answer, makes sense to me